internal/handler/request/ticket.go

```diff
@@ -0,0 +1,5 @@
+package request
+
+type UpdateTicket struct {
+	Price int64 `json:"price"`
```

Comment:
Need to double check with the initial task, but logically flight ID could also be updated. Please check and reply whether that is the case.

Reply:
didn't found any info about that in requirements

Reply:
Nvm then